### PR TITLE
Remove note about MTR

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [MTA-3844](https://issues.redhat.com/browse/MTA-3844)

### CODE REVIEW

File: docs/topics/mta-about-the-intro-to-mta-guide.adoc [hunk 1]
```diff
@@ -6,16 +6,11 @@
 [id="mta-about-the-intro-to-mta-guide_{context}"]
 = Introduction to the {ProductName}
 
-{ProductFullName} is a set of tools that you can use to accelerate large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. {ProductShortName} looks for common resources and known trouble spots when migrating applications. It provides a high-level view of the technologies used by the application. MTA also generates a detailed report that evaluates a migration or modernization path. By using this report, you can estimate the effort required for large-scale projects and reduce the work involved. 
+{ProductFullName} is a set of tools that you can use to accelerate large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. {ProductShortName} looks for common resources and known trouble spots when migrating applications. It provides a high-level view of the technologies used by the application. MTA also generates a detailed report that evaluates a migration or modernization path. By using this report, you can estimate the effort required for large-scale projects and reduce the work involved.
```
Comment (mpershina) [inline docs/topics/mta-about-the-intro-to-mta-guide.adoc:9]: For consistency, hehe :-D

```suggestion
{ProductFullName} is a set of tools that you can use to accelerate large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. {ProductShortName} looks for common resources and known trouble spots when migrating applications. It provides a high-level view of the technologies used by the application. {ProductShortName} also generates a detailed report that evaluates a migration or modernization path. By using this report, you can estimate the effort required for large-scale projects and reduce the work involved.
```

**Comment by istein1** [comment]
LGTM

**Comment by anarnold97** [comment]
@raoradhika - please can you merge this for MTA 7.1.1

I think it is an important enough removal to require an immediate fix

Masha and Ilanit have reviewed

**Comment by raoradhika** [comment]
done

**Pull request merged**